Makefile

```diff
@@ -47,9 +47,9 @@ wait-and-validate:
 	@echo "Validating"
 	SKIP_TESTS="$(SKIP_TESTS)" DONT_FOCUS=true TEST_SUITES="validationsuite" hack/run-functests.sh
 
-functests-on-ci: init-git-submodules feature-deploy-on-ci functests
+functests-on-ci: sync-git-submodules feature-deploy-on-ci functests
 
```

Comment:
are these commits ededcf124f2db2a3bcefddc4898f64a672700fa4,3cbe1bdd9bac842d0b99368be5f7a175711f667a related to rpm lockfile updates in anyway?  if not, can we have a different cleanup PR to hold them and likely merge it sooner?